source/_layouts/footer.blade.php

```diff
@@ -43,6 +43,10 @@
             </div>
         </div>
     </div>
+
+    <!-- Scroll Top -->
+    <a href="#" id="scroll-top" class="scroll-top d-flex align-items-center justify-content-center"><i class="bi bi-arrow-up-short"></i></a>
```

Comment:
Já tem o script que é executado ao clicar no elemento com id `scroll-top`?
Este botão foi movido de outro lugar?

Reply:
Yes, this block of code was copied. I was test and forget to remove.